src/elements/content-sidebar/activity-feed/icons/IconActivityFeedEmptyState.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import AccessibleSVG from '../../../../icons/accessible-svg';
+
+type Props = {
+    className?: string;
+    height?: number;
+    width?: number;
+};
+
+const IconActivityFeedEmptyState = ({ className = '', width = 140, height = 140 }: Props): JSX.Element => (
```

Comment:
Is this considered to be an icon or an illustration? Seems like the latter.

Reply:
Good point, will rename